validator/src/main.rs

```diff
@@ -411,85 +411,6 @@ fn get_cluster_shred_version(entrypoints: &[SocketAddr]) -> Option<u16> {
     None
 }
 
-fn platform_id() -> String {
-    format!(
-        "{}/{}/{}",
-        std::env::consts::FAMILY,
-        std::env::consts::OS,
-        std::env::consts::ARCH
-    )
-}
-
-#[cfg(target_os = "linux")]
-fn check_os_network_limits() {
-    use {solana_metrics::datapoint_warn, std::collections::HashMap, sysctl::Sysctl};
-
-    fn sysctl_read(name: &str) -> Result<String, sysctl::SysctlError> {
-        let ctl = sysctl::Ctl::new(name)?;
-        let val = ctl.value_string()?;
-        Ok(val)
-    }
-    let mut check_failed = false;
-
-    info!("Testing OS network limits:");
-
-    // Reference: https://medium.com/@CameronSparr/increase-os-udp-buffers-to-improve-performance-51d167bb1360
-    let mut recommended_limits: HashMap<&str, i64> = HashMap::default();
-    recommended_limits.insert("net.core.rmem_max", 134217728);
-    recommended_limits.insert("net.core.rmem_default", 134217728);
-    recommended_limits.insert("net.core.wmem_max", 134217728);
-    recommended_limits.insert("net.core.wmem_default", 134217728);
-    recommended_limits.insert("vm.max_map_count", 1000000);
-
-    // Additionally collect the following limits
-    recommended_limits.insert("net.core.optmem_max", 0);
-    recommended_limits.insert("net.core.netdev_max_backlog", 0);
-
-    let mut current_limits: HashMap<&str, i64> = HashMap::default();
-    for (key, _) in recommended_limits.iter() {
-        let current_val = match sysctl_read(key) {
-            Ok(val) => val.parse::<i64>().unwrap(),
-            Err(e) => {
-                error!("Failed to query value for {}: {}", key, e);
-                check_failed = true;
-                -1
-            }
-        };
-        current_limits.insert(key, current_val);
-    }
-
-    for (key, recommended_val) in recommended_limits.iter() {
-        let current_val = *current_limits.get(key).unwrap();
-        if current_val < *recommended_val {
-            datapoint_warn!("os-config", (key, current_val, i64));
-            warn!(
-                "  {}: recommended={} current={}, too small",
-                key, recommended_val, current_val
-            );
-            check_failed = true;
-        } else {
-            datapoint_info!("os-config", (key, current_val, i64));
-            info!(
-                "  {}: recommended={} current={}",
-                key, recommended_val, current_val
-            );
-        }
-    }
-    datapoint_info!("os-config", ("platform", platform_id(), String));
-
-    if check_failed {
-        datapoint_warn!("os-config", ("network_limit_test_failed", 1, i64));
```

Comment:
do we want to remove this warn of failure?

Reply:
good catch, replaced

Reply:
@jbiseda did you put it back and not push a new commit?